plugins/BEdita/Core/src/Model/Action/UpdateRelatedObjectsAction.php

```diff
@@ -65,4 +68,27 @@ protected function prepareData(array $data)
 
         return compact('entity', 'relatedEntities') + $data;
     }
+
+    /**
+     * Setup `priority` on `_joinData`.
+     * If relation is inverse switch `priority` and `inv_priority`.
+     *
+     * @param array $data Action data.
+     * @return void
+     */
+    protected function setupPriority(array &$data): void
+    {
+        if ($this->Association->getForeignKey() !== 'right_id') {
```

Comment:
As above we could have a method in `RelatedTo` association to know if it's an inverse relation.